### PR TITLE
Add ESLint

### PR DESCRIPTION
**Pull request by danielbayerlein**
See #13

### CODE REVIEW

**Comment by danielbayerlein** [comment]
@chrishelgert Can you please review my changes?

**Comment by chrishelgert** [comment]
Sure, will look later @ it :)

File: __tests__/helpers/on-headers-listener.spec.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ describe('helpers', () => {
       const spans = defaultConfig.spans;
 
       beforeEach(() => {
-        spans.forEach((span) => {
+        spans.forEach((currentSpan) => {
```
Comment (chrishelgert) [inline __tests__/helpers/on-headers-listener.spec.js]: should have a different meaning - maybe make spans a let and use map to iterate over the spans and clear the responses:

```javascript
let spans = defaultConfig.spans;
// ....
spans = spans.map((currentSpan) => {
  const span = currentSpan;
  span.responses = [];
  return span;
});
// ...

File: examples/server.js [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
-  register: require('hapijs-status-monitor')
-});
+server.register({ register: hapijsStatusMonitor });
 
 // Add the "/return-status/{statusCode}" route
 server.route({
   method: 'GET',
   path: '/return-status/{statusCode}',
-  handler: function (request, reply) {
+  handler: function handler(request, reply) {
```
Comment (chrishelgert) [inline examples/server.js]: why no arrow-function and parse the statusCode initial?
```javascript
handler: (request, reply) => {
  const statusCode = parseInt(request.params.statusCode, 10);
  return reply(statusCode).code(statusCode);
};

File: src/helpers/socket-io-init.js [hunk 3]
```diff
@@ -16,7 +14,8 @@ module.exports = (server, spans) => {
       });
     });
 
-    spans.forEach((span) => {
+    spans.forEach((currentSpan) => {
```
Comment (chrishelgert) [inline src/helpers/socket-io-init.js]: same as above with map or?

File: src/middleware-wrapper.js [hunk 4]
```diff
@@ -5,50 +5,50 @@ const onHeadersListener = require('./helpers/on-headers-listener');
 const socketIoInit = require('./helpers/socket-io-init');
 
 // hapi.js plugin register function
-var middlewareWrapper = function (server, options, next) {
-  options = validate(options);
+const middlewareWrapper = function middlewareWrapper(server, options, next) {
```
Comment (chrishelgert) [inline src/middleware-wrapper.js]: why no arrow-function?

Comment (danielbayerlein) [inline src/middleware-wrapper.js]: Done

File: examples/server.js (hunk 2 above)
Comment (danielbayerlein) [inline examples/server.js]: Done

File: __tests__/helpers/on-headers-listener.spec.js (hunk 1 above)
Comment (danielbayerlein) [inline __tests__/helpers/on-headers-listener.spec.js]: Done

**Comment by danielbayerlein** [comment]
@chrishelgert Thanks for the review 👍

**Pull request merged**